etc/dockerfiles/stack-build/lts-15.0/Dockerfile

```diff
@@ -0,0 +1,89 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="manny@fpcomplete.com"
+
+ARG GHC_VERSION=8.8.2
+ARG LTS_SLUG=nightly-2020-02-11
+ARG PID1_VERSION=0.1.2.0
+ARG STACK_VERSION=2.1.3
+ARG CUDA_VERSION=10.0
+ARG JVM_PATH=/usr/lib/jvm/java-8-openjdk-amd64
+ARG LLVM_VERSION=7.0
```

Comment:
In this case, I think it should be set to `7` since the directory is `/usr/lib/llvm-7/include`.  In fact I would recommend to switching to the approach used for the JVM, and making this an `LLVM_PATH` variable instead, set to `/usr/lib/llvm-7`, and then adjust the way it's used when setting the `CPATH` environment variable below.